city_scrapers/spiders/newnj_nbe.py

```diff
@@ -0,0 +1,142 @@
+import re
+from datetime import datetime, time
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class NewnjNbeSpider(CityScrapersSpider):
+    name = "newnj_nbe"
+    agency = "Newark Board of Education"
+    timezone = "America/New_York"
+    start_urls = ["https://www.nps.k12.nj.us/board-of-education/meetings/"]
+
+    def parse(self, response):
+        """
+        Most of the meetings details are provided in a detail page.
+        Though the each meetings' location address varies and is taken
+        from the main page and passed as a meta value to the followed
+        callback method.
+        """
```

Comment:
**praise:** This is helpful context for understanding how this spider is getting the job done. Thanks for including!